code/jobs/CreateEnvJob.php

```diff
@@ -28,36 +28,8 @@ public function tearDown() {
 	 *
 	 */
 	public function perform() {
-		echo "[-] CreateEnvJob starting" . PHP_EOL;
-		$log = new DeploynautLogFile($this->args['logfile']);
-		$project = $this->DNData()->DNProjectList()->filter('Name', $this->args['projectName'])->first();
 		// This is a bit icky, but there is no easy way of capturing a failed deploy by using the PHP Resque
 		try {
-			// Disallow concurrent deployments (don't rely on queuing implementation to restrict this)
-			// Only consider deployments started in the last 30 minutes (older jobs probably got stuck)
-			$running = DNCreateEnvironment::get()
-				->filter(array(
-					'ProjectID' => $project->ID,
-					'Status' => array('Queued', 'Started'),
-					'Created:GreaterThan' => strtotime('-30 minutes')
-				))
-				->exclude('ID', $this->args['createID']);
-
-
-			if($running->count()) {
-				$existing = $running->first();
-				$log->write(sprintf(
-					'[-] Error: another create job is in progress (started at %s by %s)',
```

Comment:
Nope, you should be able to create multiple envs at once (see RF fix). Use case is where the customer creates the new stack and both envs need to be created. Given this will usually be their first interaction with platform, I wanted the creation to be quick (half the time).